src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerStatus = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        {age
+          ? (
+            <p className="Person__age">
+              {`I am ${age}`}
+            </p>
+          )
+          : null}
```

Comment:
i think it's better to implement it with `&&` operator
```suggestion
        {age
          && (
            <p className="Person__age">
              {`I am ${age}`}
            </p>
          )}
```

Reply:
yessss!!! Saw in your work, you right!